website/content/docs/connect/observability/grafanadashboards/consulk8sdashboard.mdx

```diff
@@ -0,0 +1,80 @@
+---
+layout: docs
+page_title: Dashboard for Consul k8s control plane metrics
+description: >-
+  This documentation provides an overview of the Consul K8s Dashboard
+---
+
+# Consul k8s monitoring (Control Plane) dashboard
+
+### Number of Consul servers
+
+- **Grafana query:** `consul_consul_server_0_consul_members_servers{pod="consul-server-0"}`
+- **Description:** Displays the number of Consul servers currently active. This metric provides insight into the cluster's health and the number of Consul nodes running in the environment.
```

Comment:
```suggestion
- **Grafana query:** `consul_consul_server_0_consul_members_servers{pod="consul-server-0"}`
- **Description:** Displays the number of Consul servers currently active. This metric provides insight into the cluster's health and the number of Consul nodes running in the environment.
```
This only retrieves the metric from the pod named `consul-server-0`. Is it possible to modify this so that the value is retrieved from any available server, or the Raft leader?